src/Common/src/Microsoft/Win32/SafeHandles/SafeUnicodeStringHandle.cs

```diff
@@ -24,13 +24,16 @@ public SafeUnicodeStringHandle(string s)
         public unsafe SafeUnicodeStringHandle(ReadOnlySpan<char> s)
             : base(IntPtr.Zero, ownsHandle: true)
         {
-            int cch = checked(s.Length + 1);
-            int cb = checked(cch * sizeof(char));
-            handle = Marshal.AllocHGlobal(cb);
+            if (s != default)
```

Comment:
Assuming you're drawing a distinction here between `s != default` and `s.Length > 0`, it'd be good to add a comment to that effect.